src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin animation-hover($property, $value) {
+  transition: $property $time-for-animation;
+
+  &:hover {
+    #{$property}: $value;
```

Comment:
Interpolation (`#{}`) is not necessary for the property name in Sass. You can simply use `$property: $value;` instead.